test/src/main/scala/scalarules/test/scala_import/BUILD

```diff
@@ -36,14 +28,6 @@ scala_specs2_junit_test(
     deps = [":guava_and_commons_lang"],
 )
 
-scala_specs2_junit_test(
-    name = "scala_import_exposes_file_jars",
-    size = "small",
-    srcs = ["ScalaImportExposesFileJarsTest.scala"],
-    suffixes = ["Test"],
-    deps = [":relate"],
-)
-
```

Comment:
this is about testing core bazel functionality introduced [here](https://github.com/mateuszkuta256/rules_scala/commit/efffa526a31724803532b46e659a7ea3240b9509)
thus I think it's not crucial to have such a test. Dropping it allows us to get rid of .jar file

Reply:
@mateuszkuta256 this tests if `scala_import` is working correctly, why do you think it is not needed?

Reply:
this test uses the "relate_2.11-2.1.1.jar" file that I want to get rid of because of security alerts
I replaced jar files with "scala_import" wherever possible, but for this case it's not possible, because the test is exactly about importing .jar file
according to to git history it was introduced along with [this](https://github.com/mateuszkuta256/rules_scala/commit/efffa526a31724803532b46e659a7ea3240b9509) single line change: `"jars": attr.label_list(allow_files=True)`
looks to me like testing core bazel functionality rather than scala import rule itself, so I think we can drop the test so we could drop  "relate_2.11-2.1.1.jar" too

Reply:
what will happen if someone accidentally removes `allow_files=True`from `scala_import` implementation?

Reply:
in such a case the project won't build anymore
[repository def](https://github.com/mateuszkuta256/rules_scala/blob/12d60d203591d92572c812f345b45babff688230/third_party/repositories/repositories.bzl#L59) leads to [this](https://github.com/mateuszkuta256/rules_scala/blob/0391ef4e43fecac5b63518082294e2151f74aeb4/scala/scala_maven_import_external.bzl#L202) clause which generates BUILD files:
```
# DO NOT EDIT: generated by jvm_import_external()

load("@io_bazel_rules_scala//scala:scala_import.bzl", "scala_import")

package(default_visibility = ["//visibility:public"])

licenses(["notice"])

scala_import(
    name = "io_bazel_rules_scala_scala_xml",
    jars = ["scala-xml_2.12-1.2.0.jar"],
    srcjar = "scala-xml_2.12-1.2.0-src.jar",
)
```
`jars = ["scala-xml_2.12-1.2.0.jar"]` does not compile then
of course would be great to have an explicit test... but I lack an idea how to do that without putting .jar into the codebase

Reply:
such test may look redundant because we already implicitly test that code in Rules Scala by depending on that functionality in our codebase ourselves, but:
- it's easier to develop such features when there's a small isolated test case ava
- at some point we may change internal dependency on `scala_import` and it won't be implicitly tested anymore

Reply:
thx! your solution works well. I also removed the very last `thirft3_scrooge.jar`

Reply:
@mateuszkuta256 please push your changes

Reply:
ah, pushed to another branch